### ISSUE TITLE
The `incorrect-bytes-format` works incorrectly

### ISSUE THREAD

**Issue by maxonfjvipon** [objectionary/lints#104]
1. It does not fail on `00-00-` bytes 
2. it works only with top level objects because of this:
```xml
<xsl:template match="/">
  <defects>
    <xsl:apply-templates select="objects/o"/>
  </defects>
</xsl:template>
```
3. It fails on correct bytes in different places:
```xml
<o name="correct-inners" line="42">
  <o base="first">
    <o base="org.eolang.bytes">3F-F0-00-00-00-00-00-00</o>
  </o>
  <o base="second">
    <o base="org.eolang.bytes">00-00-00-00-00-00-00-00</o>
  </o>
</o>
```
The message here: Incorrect bytes format: "3F-F0-00-00-00-00-00-00 00-00-00-00-00-00-00-00".

**Comment by yegor256** [comment]
@rultor release, tag is `0.0.20`
